pkgs/by-name/ta/tabby-agent/package.nix

```diff
@@ -0,0 +1,69 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  nix-update-script,
+  nodejs,
+  pnpm,
+  wrapGAppsHook3,
+}:
+stdenv.mkDerivation (finalAttrs: {
```

Comment:
Have you tried using `buildNodePackage` ?

Reply:
buildNodePackage doesn't work with pnpm lockfiles so we use mkDerivation.

Reply:
Ok, very clear !